assets/locales/de.json

```diff
@@ -536,15 +598,26 @@
     "enrollment_status": "Enrollment Status"
   },
   "security": {
-    "security": "Security"
+    "report": "Sicherheits-Report",
+    "security": "Security",
+    "gatekeeper": "",
```

Comment:
Ok, on it. I thought that only strings that have a translated word had to be filled in, sorry! Will send new PR.